CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
Presumably backing it would break the verify script.

@david-mcmahon Could you PTAL?